<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_round">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>109</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>круг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ss">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>старт/стоп</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clear">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>109</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>631</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Время</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_t">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>109</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>тема</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_round</sender>
   <signal>clicked()</signal>
   <receiver>tbr</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>tbr</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ss</sender>
   <signal>toggled(bool)</signal>
   <receiver>lb_time</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ss</sender>
   <signal>toggled(bool)</signal>
   <receiver>pb_round</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ss</sender>
   <signal>toggled(bool)</signal>
   <receiver>pb_clear</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>lb_time</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
